docs/en/advanced_tutorials/test_time_augmentation.md

```diff
@@ -0,0 +1,161 @@
+# Test time augmentation
+
+Test time augmentation (TTA) is a data augmentation strategy used during the testing phase. It involves applying various augmentations, such as flipping and scaling, to the same image and then merging the predictions of each augmented image to produce a more accurate prediction. To make it easier for user to use TTA, MMEngine provides [BaseTTAModel](mmengine.model.BaseTTAModel) class, which allows users to implement different TTA strategies by simply extending the `BaseTTAModel` class according to their needs.
+
+The core implementation of TTA is usually divided into two parts：
+
+1. Data augmentation：This part is implemented in MMCV, see the api docs [TestTimeAug](mmcv.transform.TestTimeAug) for more information.
+2. Merge the predictions：The subclasses of `BaseTTAModel.test_step` will merge the predictions of enhanced data to a more accurate prediction.
```

Comment:
```suggestion
2. Merge the predictions: The subclasses of `BaseTTAModel.test_step` will merge the predictions of enhanced data to a more accurate prediction.
```

Reply:
The sentence "The subclasses of `BaseTTAModel.test_step`" is not consistent with Chinese documentation.